src/traceViewer/web/ui/sourceTab.ts

```diff
@@ -45,12 +45,12 @@ export class SourceTab implements Tab {
     }
     const { action } = actionEntry;
     const frames = action.stack!.split('\n').slice(1);
-    const frame = frames.filter(frame => !frame.includes('playwright/lib/client/'))[0];
+    const frame = frames.filter(frame => !frame.includes('playwright/lib/') && !frame.includes('playwright/src/'))[0];
```

Comment:
There are a lot of gotchas here. Eventually should probably move to a third party library. You probably want to avoid frame lines that are for internal node stuff.

Reply:
Yeah...